### ISSUE TITLE
Creation of notes does not work for entries (PHP 8.3.6)

### ISSUE THREAD

**Issue by KiralyCraft** [mkucej/i-librarian-free#98]
I'm trying to add notes to an article on a system with Apache, php-fpm and PHP 8.3.6

It seems to give this error, and no notes are saved:

```
E_DEPRECATED: Creation of dynamic property HTMLPurifier_Lexer_DOMLex::$_entity_parser is deprecated
```

I'm currently at commit `f94037241c1827a1b6ac8c555edd09f367032d3b`

**Comment by KiralyCraft** [comment]
I seem to have fixed it. I downloaded version `0.14.15` of HTMLPurifier from [here](http://htmlpurifier.org/download), and replaced the contents of `i-librarian-free/classes/libraries/htmlpurifier` with the contents of the `library` folder from the archive.

I only used the `HTMLPurifier` folder, the `HTMLPurifier.php` file and the `HTMLPurifier.includes.php` file. I renamed `HTMLPurifier.php` to `something_else.php`, renamed `HTMLPurifier.includes.php` to `htmlpurifier.php` and then edited it to `require 'something_else.php'` instead of `require 'HTMLPurifier.php`.

It seems to save notes correctly now.
